github-org/action.yaml

```diff
@@ -29,31 +25,26 @@ inputs:
       Sets the score threshold for scans. Scores that fall below the threshold will exit 1. (default "0" - job continues regardless of the score returned by a scan).
     default: "0"
     required: false
-  service-account-credentials:
-    description: "Base64 encoded service account credentials used to authenticate with Mondoo Platform"
-    required: true
   is-cicd:
     description: "Automatically detects the GitHub Action Runs and report results into the CI/CD view"
-    default: true
+    default: "true"
```

Comment:
why is this in double quotes? isn't it a `bool`?

Reply:
Fixed